R/continuous_wrap.R

```diff
@@ -1,71 +0,0 @@
-#' Calculate margins and se for continuous variables
-#'
-#' For one set of transformed covariates (not including the variable of interest),
-#' calculate the predicted level and se for the variable of interest.
-#'
-#' @param df_trans data.frame, should already be transformed for variables not related to
-#' the variable of interest
-#' @param var_interest the variable of interest
-#' @param at_var_interest levels of variable of interest
-#' @param model model
-#' @param type either effects or levels, defaults to levels
-#' @param vcov_mat variance-covariance matrix, defaults to NULL
-#' @return dataframe of formatted output
-#' @importFrom stats coef
-#' @export
-#'
-#' @examples
-#' data(mtcars)
-#' mtcars$gear <- factor(mtcars$gear)
-#' mm <- glm(vs ~ gear + mpg * disp, mtcars, family = 'binomial')
-#' continuous_wrap(mm$model, var_interest = 'mpg',
-#'                 at_var_interest = c(15, 21), model = mm,
-#'                 vcov_mat = vcov(mm))
-continuous_wrap <- function(df_trans, var_interest, model,
-                            at_var_interest = NULL, type = 'levels',
-                            vcov_mat = NULL){
-
-  stopifnot(is.data.frame(df_trans),
-            is.character(var_interest),
-            var_interest %in% names(df_trans),
-            type %in% c('effects', 'levels'))
-
-  df_levels <- at_transforms(
-    df_trans, gen_at_list(df_trans, var_interest, at_var_interest))
-
-  # Get predicted values and covariates
-  cov_preds <- lapply(df_levels, function(x)
-    predict_modelmat(model = model, transformed_df = x))
-
-  # calculate predictions
-  preds <- sapply(cov_preds, function(x){ mean(x$pred_resp) })
-
-  # if covariates are dropped from the model,
-  # remove those columns from cov_preds
-  for(i in 1:length(cov_preds)){
-    cov_preds[[i]]$covar <- cov_preds[[i]]$covar[, !is.na(coef(model))]
-  }
-
-  # calculate jacobian
-  jacobs <- do.call(rbind, lapply(cov_preds, function(x){
-    if(type == 'levels'){
-      calc_jacob(
```

Comment:
So this exists here: https://github.com/anniejw6/modmarg/blob/34c0ff3fe259859239fccb10374bd70f38516bee/R/pred_se_wrap.R#L57-L62